### PR TITLE
Add examples to search documentation

### PR DESCRIPTION
**Pull request by Allerter**
Added two examples to Search API with queries that include filter fields, and added the link to the Search for an Item page in the official docs which provides more info on how to write queries.

### CODE REVIEW

File: tekore/_client/api/search.py [hunk 1]
```diff
@@ -80,6 +80,12 @@ def search(
             tracks, = spotify.search('monty python')
             artists, = spotify.search('sheeran', types=('artist',))
             albums, tracks = spotify.search('piano', types=('album', 'track'))
+            spotify.search('gold album:boba artist:abba', types=('track',))
+            spotify.search('bob year:1980-2020', types=('show',))
+
+        .. note::
+            You can narrow down search results by specifying filed filters (e.g. year range, genre).
```
Comment (felix-hilden) [inline tekore/_client/api/search.py:87]: Also typo here, 'Filed' = 'Field'?

Comment (Allerter) [reply tekore/_client/api/search.py:87]: Yes, that's a typo. Meant to write "field".

**Comment by felix-hilden** [comment]
I'm sorry the typo is still there 😅

**Comment by Allerter** [comment]
> I'm sorry the typo is still there  but otherwise this is exactly it!

Really fixed it this time :sweat_smile:

**Pull request merged**